### DISCUSSION TITLE
Integrate Slack Incoming Web-hooks

### DISCUSSION THREAD

**Discussion by vijopv83** [ludeknovy/jtl-reporter#180] [Ideas]
Slack Incoming web-hooks enables us to share information from external sources with other Slack Users.
Will be great if we can Push Notification messages to Slack too...

**Comment by vijopv83** [comment]
Sharing a sample Body for Slack Incoming Web-hooks:

`{
	"blocks": [
		{
			"type": "header",
			"text": {
				"type": "plain_text",
				"text": "JTL Reporter"
			}
		},
		{
			"type": "header",
			"text": {
				"type": "plain_text",
				"text": "Performance Report for scenario: Login"
			}
		},
		{
			"type": "divider"
		},
		{
			"type": "section",
			"fields": [
				{
					"type": "mrkdwn",
					"text": "*Error Rate*\n0 %"
				}
			]
		},
		{
			"type": "section",
			"fields": [
				{
					"type": "mrkdwn",
					"text": "*90% percentile*\n48 ms"
				}
			]
		},
		{
			"type": "section",
			"fields": [
				{
					"type": "mrkdwn",
					"text": "*Throughput*\n37.16 reqs/s"
				}
			]
		},
		{
			"type": "section",
			"fields": [
				{
					"type": "mrkdwn",
					"text": "*Duration*\n34.97 min"
				}
			]
		}
	]
}`

**Comment by vijopv83** [comment]
<img width="445" alt="Slack" src="https://user-images.githubusercontent.com/65507674/181604123-24c621c6-e374-4d45-92b9-23a37b3f279e.png">
Sharing a sample of the above Message in Slack.

**Reply by ludeknovy** [reply]
Implemented https://github.com/ludeknovy/jtl-reporter/issues/181
